source/usage-examples/command.txt

```diff
@@ -56,7 +56,7 @@ Example
    The JavaScript and TypeScript code snippets above are identical. There are no
    TypeScript specific features of the driver relevant to this use case.
 
-When you run the preceding command, you should see the following output:
+Running the preceding command, you see the following output:
```

Comment:
S: I think the copy on all of the usage examples should match:
```suggestion
Running the preceding example results in the following output:
```

Reply:
Leaving as is to match the other occurences.